Day1/Shubham_Nagaria_Pascals_Triangle.cpp

```diff
@@ -0,0 +1,28 @@
+// Problem Statement - https://leetcode.com/problems/pascals-triangle
+// Problem - Pascal's Triangle
+class Solution {
+public:
+    vector<vector<int>> generate(int numRows) {
```

Comment:
Try to separate code and comments.
Comments are basically stating the steps and the code is doing what it is doing. Comments before method as method doc is better and not within the code. 
It's very difficult to change the code if comments are spread everywhere. Separate both please.